doc/source/index.rst

```diff
@@ -1,9 +1,33 @@
-Welcome to SUSE Containerized OpenStack
-=======================================
+Welcome to SUSE Containerized OpenStack Technology Preview
+==========================================================
 
-The socok8s project automates SUSE Containerized OpenStack provisioning
-and lifecycle management on SUSE Container as a Service Platform (CaaSP) and
-SUSE Enterprise Storage (SES), using Airship, shell scripts, and Ansible playbooks.
+The socok8s project automates SUSE Containerized OpenStack provisioning and
+lifecycle management on SUSE Container as a Service Platform (CaaSP) and
+SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
+and Ansible playbooks.
+
+As a technology preview, the included features are not supported. They may be
+functionally incomplete, unstable or in other ways not suitable for production
+use. They are mainly included for customer convenience and give customers a
+chance to test new technologies within an enterprise environment.
+
```

Comment:
We need to rework the paragraph above.  

Suggested rework.

As a technology preview, this is not meant for production implementation and a typical support subscription for the technology preview is not being offered.  This technology preview gives customers and partners the ability to test and see the new technologies that will be offered with full support in future releases.  As a caution some of the functionality currently in our generally available SUSE OpenStack Cloud product will not be available in this tech preview.